example/15_grouped_gemm/grouped_gemm_xdl_fp16.cpp

```diff
@@ -168,7 +168,7 @@ int main(int argc, char* argv[])
         }
     }
 
-    for(int i = 0; i < gemm_shapes.size(); i++)
+    for(int i = 0; i < ck::type_convert<int>(gemm_shapes.size()); i++)
```

Comment:
Consider `size_t i`. And all following similar cases

Reply:
Addressed by [57f2d3c](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/57f2d3c31e163a8525856fb3b1cae2cb09ddbff9)